knative-operator/hack/update-manifests.sh

```diff
@@ -68,3 +68,9 @@ git apply "$root/knative-operator/hack/001-broker-config-tracing.patch"
 
 # For now we remove the CRDs, since the "broker" does not yet do anything with them
 git apply "$root/knative-operator/hack/003-broker-remove-duplicated-crds.patch"
+
+# For now we need to add prober env variables.
+git apply "$root/knative-operator/hack/004-kafka-broker-prober-env.patch"
+
+# For now we need to add broker read access to the webhook.
+git apply "$root/knative-operator/hack/005-kafka-broker-webhook-role.patch"
```

Comment:
doing the patches, for "matching" upstream manifest seems to exactly what we want. 

Thanks for going that route after we discussed